lib/Twig/Extension/Core.php

```diff
@@ -852,6 +852,34 @@ function twig_in_filter($value, $compare)
     return false;
 }
 
+/** 
+ * Returns a trimmed string.
+ * 
+ * @param string $string
+ * @param string $characterMask
+ * @param string $mode
+ * @return string
+ */
+function twig_trim_filter($string, $characterMask = null, $mode = 'both')
+{
+    if (empty($characterMask)) {
+        $characterMask = " \t\n\r\0\x0B";
+    }
+    switch ($mode) {
+        case 'left':
+            $trimmed = ltrim($string, $characterMask);
```

Comment:
Is it not better practice to have only one "Return" in a function rather than multiple?